cypress/integration/Basic Flows/project.spec.js

```diff
@@ -0,0 +1,41 @@
+/// <reference types="cypress" />
+
+context('Projects', () => {
+  beforeEach(() => {
+    cy.loginWithUI()
+  })
+
+  it('user can create a new project', () => {
+    let projectTitle = "Fake Title"
+    let projectAcronym = "FT"
+
+    cy.visit(Cypress.env('baseUrl') + '/packages/create')
+
+    // filling out form
+    cy.get('#id_title').type(projectTitle)
+    cy.get('#id_acronym').type(projectAcronym)
+    cy.get('.usa-radio__label').contains('CMS AWS Commercial East-West').click()
+
+    cy.get('.usa-radio__label').contains('Low').click()
+    cy.get('#next-button').click()
+
+    //in the future this will automatically redirect to the correct page
+    // this is a stand in until we update routing
+    cy.visit(Cypress.env('baseUrl') + '/packages')
+    cy.get('.usa-card__heading').should('contains.text', projectTitle)
```

Comment:
Hmm, so I have a lot of projects created by my user in local and when this one is created, it orders it last on the list. Since the project is on another page, the test fails for me. This may not be a "right now" problem, but a couple things we should probably think about are: deciding how we want to order projects on the projects page, deciding what we want to do about our local test databases when running cypress